doc/changelog/03-notations/13519-primitiveArrayNotations.rst

```diff
@@ -0,0 +1,8 @@
+- **Added:**
+  :cmd:`Number Notation` and :cmd:`String Notation`
+  commands now support primitive floats and primitive arrays
+  as well as the type constants of all primitive objects as parameter of inductives and arrays
```

Comment:
```suggestion
  :cmd:`Number Notation` and :cmd:`String Notation`
  commands now support primitive floats, primitive arrays
  and type constants of primitive objects as parameters of inductives and arrays
```

I don't understand the `as parameters of inductives and arrays`, maybe `as` should be `for`?  And it's unclear how `parameters` and `inductives and arrays` relate to these syntax-oriented commands.  Are you referring to _printing_ inductives and arrays?

Reply:
I rephrased the whole changelog entry.
I want to communicate that:
-objects whose type is primitive float and primitive arrays can now be printed/parsed, while before this was only there for primitive integers.
-The constants that are the type of those objects (`Numbers.Cyclic.Int63.Int63.int` etc) are now allowed inside terms that get parsed/printed, which is for example needed when constructing polymorphic datatypes or arrays containing primitive values.
Would you mind to give feedback on the new phrasing?